docs/examples/combined/calendar-agent.mdx

```diff
@@ -141,16 +155,21 @@ description: "This project is an example which uses Composio to seamlessly conve
     <CodeGroup>
         ```javascript Models and Tools
         // Initialize the language model
-        const llm = new ChatOpenAI({ model: "gpt-4-turbo" });
+        const llm = new ChatOpenAI({ model: "gpt-4" });
 
         // Define tools for the agents
         // We are using Google calendar tool from composio to connect to our calendar account.
-        const composioToolset = new LangchainToolSet({
-            apiKey: process.env.COMPOSIO_API_KEY
-        });
-        const tools = await composioToolset.getTools({
-            actions: ["googlecalendar_create_event", "googlecalendar_list_events"]
-        });
+        try {
```

Comment:
Good addition of try-catch block for error handling in the JavaScript example. This matches the Python implementation and provides better error handling for users.